internal/tfengine/tfengine_test.go

```diff
@@ -17,10 +17,14 @@ package tfengine
 import (
 	"io/ioutil"
 	"os"
+	"os/exec"
 	"path/filepath"
+	"regexp"
 	"testing"
 )
 
+var backendRE = regexp.MustCompile(`(?s)backend "gcs" {.*?}`)
```

Comment:
I would leave a comment linking to somewhere explaining the `.*?`. I don't think it's obvious since it's a rarely-encountered regex thing. Also maybe point out that by default it works in multiline mode instead of line-by-line.